spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -83,6 +83,33 @@
       end
     end
 
+    describe "admin weight calculated fees" do
+      context 'order with 2 line items' do
+        let(:admin_fee) { create(:enterprise_fee, calculator: Calculator::Weight.new(preferred_per_kg: 1),
+          enterprise: order_cycle.coordinator, fee_type: 'admin') }
+
+        before do
+          product_fee.variants.first.update_attributes(unit_value: '2000.0')
+          product_tax.variants.first.update_attributes(unit_value: '5000.0')
+
+          add_enterprise_fee admin_fee
+
+          cart_service = CartService.new(order)
+          cart_service.populate(variants: { product_fee.variants.first.id => 3, product_tax.variants.first.id => 5 })
```

Comment:
The quantity of 5 here really confused me because the cart total was only computing for 3 units. If the product factory's `on_hand` value of 3 for `product_tax` is important in this test, what do you think of specifying this attribute when defining `product_tax`?